backup/bin/setup-barman

```diff
@@ -113,6 +114,20 @@ def initial_backup(conf):
         subprocess.run(["barman", "check", nm], check=True)
 
 
+def setup_cron(conf):
+    print("Setting up cron jobs", flush=True)
+    if len(conf) > 1:
+        raise Exception("This needs work to support > 1 db")
+    cron = """
+* * * * * root barman cron
+0 6 * * * root update-nightly {}
```

Comment:
Does this mean 6am? Not disagreeing, but why that time? Maybe put the justification in a comment?

Reply:
Alex and I were just discussing this at length.  It should be earlier most likely